ert/storage/_storage.py

```diff
@@ -34,14 +34,15 @@
 _PARAMETER_RECORD_SEPARATOR = "."
 
 
-class _NumericalMetaData(BaseModel):
+class _RecordCollectionMetaData(BaseModel):
```

Comment:
Not sure about changing this class. If you really want to change it I would say that https://github.com/equinor/ert/issues/1710 is blocking this PR.

Reply:
Well, that depends on which on is going to be merged first, #1710 does not seem to be near completion.